cns/restserver/ipam.go

```diff
@@ -181,6 +181,37 @@ func (service *HTTPRestService) GetPodIPConfigState() map[string]cns.IPConfigura
 	return service.PodIPConfigState
 }
 
+func (service *HTTPRestService) getPodIPIDByOrchestratorContexthandler(w http.ResponseWriter, r *http.Request) {
+	var (
+		resp          cns.GetPodContextResponse
+		statusCode    int
+		returnMessage string
+		err           error
+	)
+
+	statusCode = UnexpectedError
+
+	defer func() {
+		if err != nil {
```

Comment:
this will never be non-nil. this entire function can just be
```go
resp := cns.GetPodContextResponse{
    PodContext: service.GetPodIPIDByOrchestratorContext()
}

err := service.Listener.Encode(w, &resp)

logger.Response(service.Name, resp, 0, ReturnCodeToString(0), err)
```
let's discuss